Mapsui.Geometries/ArrowLine.cs

```diff
@@ -0,0 +1,196 @@
+﻿using Mapsui.Geometries.Utilities;
+using System;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Mapsui.Geometries
+{
+    /// <summary>
+    ///     An ArrowLine is an oriented line with an arrow as direction indicator 
+    /// </summary>
+    public class ArrowLine : MultiLineString
+    {
+
+        /// <summary>
+        ///     Initializes an instance of an ArrowLine
+        /// </summary>
+        public ArrowLine() : base() { }
+
+        /// <summary>
+        ///     Initializes an instance of an ArrowLine from two points, and specifications for the arrow (angle and length)
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="target"></param>
+        /// <param name="degreeAngle"></param>
+        /// <param name="arrowLength"></param>
+        public ArrowLine(Point source, Point target, double degreeAngle = 45, double arrowLength = 10)
+        {
+            double radiansAngle = Algorithms.DegreesToRadians(degreeAngle);
+            Angle = degreeAngle;
+            LineStrings = new Collection<LineString>();
+
+            LineString line = new LineString(new Point[] { source, target });
+            LineStrings.Add(line);
+
+            LineString firstArrowSide = new LineString(new Point[] { ArrowHead, ArrowExtremities(radiansAngle, arrowLength)[0] });
+            LineString secondArrowSide = new LineString(new Point[] { ArrowHead, ArrowExtremities(radiansAngle, arrowLength)[1] });
```

Comment:
ArrowExtremities is called twice: this could be stored in a variable to more efficient.